src/profile-v2/CertificateCard.jsx

```diff
@@ -27,6 +28,8 @@ const CertificateCard = ({
     audit: null,
   }[certificateType] || null;
 
+  const isMobileView = useWindowSize().width <= breakpoints.small.minWidth;
```

Comment:
export function useIsOnMobile() {
  const windowSize = useWindowSize();
  return windowSize.width <= breakpoints.small.minWidth;
}
consider creating hooks for window screens